lib/sgtn-client/util/service-util.rb

```diff
@@ -0,0 +1,99 @@
+# 
+#  Copyright 2019-2022 VMware, Inc.
+#  SPDX-License-Identifier: EPL-2.0
+#
+
+require 'multi_json'
+
+module SgtnClient
+
+  # A service util class to serve for translation API
+  class ServiceUtil
```

Comment:
"ServiceUtil" is too generic and a hodgepodge of many service methods. This can be further broken down. For example, load_offline and load_online can be in separate services as theyrequire different things.

Let's please refactor in a separate PR.